android/src/meez/devicesupport/Extension.java

```diff
@@ -54,6 +54,13 @@ public FREObject execute(FREObject[] params) throws Exception
                 return FREObject.newObject(getDeviceId());
             }
         });
+        register(new BaseFREFunction("nativeBack",new Class[]{}) {
```

Comment:
`nativeBack` doesn't seem like the best name? everything in `device-support` is native (one way or another)

`navigateBack` or something?